### ISSUE TITLE
Implement sensor details component

### ISSUE THREAD

**Issue by caseycesari** [azavea/ism-watershed-wellness-snapshot#22]
- This is just to implement the base component shown in the screenshot below.
- Updating the charts/graphics will be done after #20 is in in a separate card.

![image](https://user-images.githubusercontent.com/1042475/51708453-f0db6400-1ff1-11e9-9fbf-deb2c2e0bcaa.png)

Depends on #37
